sources/install.sh

```diff
@@ -1114,6 +1114,12 @@ function install_ysoserial() {
   add-test-command "ysoserial --help; ysoserial --help |& grep 'spring-core:4.1.4.RELEASE'"
 }
 
+function install_slipit() {
+  # Utility for creating ZipSlip archives
+  colorecho "Installing slipit"
+  pip3 install slipit
```

Comment:
Can you switch to `python3 -m pip install slipit`?
Can you also add `add-test-command`, and if you have some example commands `add-history` as well

Reply:
This tool has setup.py file. Please, can you switch pip install with pipx install ?

python3 -m pipx install slipit

Reply:
Hello, i've update the brach according to your reviews